ginga/ImageView.py

```diff
@@ -23,6 +23,8 @@
 from ginga.canvas.types.layer import DrawingCanvas
 from ginga.util import io_rgb
 
+__all__ = ['ImageViewBase']
+
```

Comment:
Related reading: http://stackoverflow.com/questions/44834/can-someone-explain-all-in-python